docs/faq.rst

```diff
@@ -0,0 +1,106 @@
+.. _FAQ:
+
+FAQ
+===
+
+.. contents:: Question Lists
+  :local:
+
+Why does the warning related to ``num_communication_per_step`` or ``backward_passes_per_step`` pop up in Bluefog optimizer?
+--------------------------------------------------------------------------------------------
+
+During your usage of Bluefog optimizer, you may encounter the following the two types of warnings.
```

Comment:
the following two types (remove the middle "the")

Reply:
Done